peachjam/templates/peachjam/layouts/main.html

```diff
@@ -17,10 +17,10 @@
     <script defer src="https://browser.sentry-cdn.com/6.18.1/bundle.min.js" integrity="sha384-XOitiDSsTajhNQgwzQ7iHMlM2Q2ODCW6aw3m1QgMVrDOqsNhFao1VjB39qXNMOZW" crossorigin="anonymous"></script>
     <script>
     window.addEventListener('load', function() {
-        Sentry.init({
+      Sentry.init({
         dsn: '{{ SENTRY_DSN_KEY }}',
         environment: '{{ SENTRY_ENVIRONMENT }}',
-        blacklistUrls: [],
+        whitelistUrls: [`${window.location.hostname}/static/js/.+.js$`]
```

Comment:
I think this must be a regex, eg:

```js
allowUrls: [new RegExp(window.location.hostname.replace(".", "\\.") + "/static/js/.+\\.js$")]
```

Reply:
I set up my own sentry instance and tested out this config. No events were sent to the sentry instance.

However, I managed to filter events using [beforeSend](https://github.com/laws-africa/peachjam/pull/664/files#r1022056945)